.github/workflows/syntax.yml

```diff
@@ -0,0 +1,50 @@
+name: syntax
+on:
+  pull_request:
+    branches:
+      - "**"
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v39
+        with:
+          files: |
+            **/*.py
+      - name: Prepare python env
+        run: |
+          python -m venv env
+          source env/bin/activate
+          pip install -U pip
+          pip install -r requirements.txt
```

Comment:
Do we really need to install everything from `requirements.txt`?

Reply:
we must install web-server dependencies for the correct work of `pylint` and `isort`, because they're unable to correctly resolve unknown dependencies

for example:
```
pylint:
albs-web-server/alws/release_planner.py:11:0: E0401: Unable to import 'immudb_wrapper' (import-error)

isort:
~/Almalinux $ env/bin/isort albs-web-server/alws/release_planner.py --diff --color --profile black --check-only --py 39
ERROR: /home/sandman/Almalinux/albs-web-server/alws/release_planner.py Imports are incorrectly sorted and/or formatted.
--- /home/sandman/Almalinux/albs-web-server/alws/release_planner.py:before	2023-09-18 18:38:49.292126
+++ /home/sandman/Almalinux/albs-web-server/alws/release_planner.py:after	2023-09-25 11:08:46.126210
@@ -8,23 +8,17 @@
 from abc import ABCMeta, abstractmethod
 from collections import defaultdict
 
-from immudb_wrapper import ImmudbWrapper
-from sqlalchemy import or_, update
-from sqlalchemy.ext.asyncio import AsyncSession
-from sqlalchemy.future import select
-from sqlalchemy.orm import selectinload
-
 from alws import models
 from alws.config import settings
 from alws.constants import (
     LOWEST_PRIORITY,
     BeholderKey,
+    BeholderMatchMethod,
     ErrataPackageStatus,
     PackageNevra,
     ReleasePackageTrustness,
     ReleaseStatus,
     RepoType,
-    BeholderMatchMethod,
 )
 from alws.crud import products as product_crud
 from alws.crud import sign_task
@@ -53,6 +47,11 @@
     get_rpm_packages_from_repository,
     get_uuid_from_pulp_href,
 )
+from immudb_wrapper import ImmudbWrapper
+from sqlalchemy import or_, update
+from sqlalchemy.ext.asyncio import AsyncSession
+from sqlalchemy.future import select
+from sqlalchemy.orm import selectinload
 
 __all__ = [
     "CommunityReleasePlanner",
```

Reply:
I see, thanks for the clarification :)